src/ExpressionEvaluator/Core/Source/Concord/Microsoft.VisualStudio.Debugger.Engine.Portable.il

```diff
@@ -6,17 +6,17 @@
 .assembly extern System.Globalization
 {
 	.publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A)
-	.ver 4:0:10:0
```

Comment:
These are reference assemblies taken from the debugger. Turns out that the debugger is targeting assemblies incompatible with Profile7. Will need to address that somehow later.